pkg/extension/workspace/example/envoy/template/expand_test.go

```diff
@@ -68,12 +68,12 @@ func TestExpand(t *testing.T) {
 		{
 			name:     "extension: config",
 			input:    `{{ .GetEnvoy.Extension.Config }}`,
-			expected: `{"@type":"type.googleapis.com/google.protobuf.StringValue","value":"{\"key\":\"value\"}"}`,
+			expected: `{"@type":"type.googleapis.com/google.protobuf.StringValue", "value":"{\"key\":\"value\"}"}`,
 		},
 		{
 			name:     "access to proto message",
-			input:    `{{ (.GetEnvoy.Extension.Code).Message.GetLocal.GetFilename }}`,
-			expected: `/path/to/extension.wasm`,
+			input:    `"{{ (.GetEnvoy.Extension.Code).Message.GetLocal.GetFilename }}"`,
+			expected: `"/path/to/extension.wasm"`,
```

Comment:
what do we need this change?

Reply:
why

Reply:
makes testing easier as we can use json assertions on all the cells

Reply:
before, this cell was the only one that wasn't valid json itow

Reply:
thanks, make sense!